lam4doc/config.py

```diff
@@ -85,11 +86,16 @@ def LAM_HTML_REPORT_TEMPLATE_LOCATION(self) -> str:
 
     @property
     def LAM_PDF_REPORT_TEMPLATE_LOCATION(self) -> str:
-        # correct value will be used when pdf implementation is ready
-        value = str(Path(__file__).parents[1] / 'templates/html')
+        value = str(Path(__file__).parents[1] / 'templates/pdf')
         self.logger.debug(value)
         return value
 
+    @property
+    def LAM_PDF_REPORT_TEMPLATE_LATEX_FILES(self) -> list:
```

Comment:
I guess this is fairly safe as long as we package the templates together with the code.